source/includes/api/_query.md

```diff
@@ -1,204 +1,512 @@
 # Query
 
-In order to retrieve data from the datasets it is possible to send queries to the API using the SQL or Feature Service languages.
+In order to retrieve data from datasets, you can send queries to the API using a syntax very similar to SQL. Using these endpoints, you can also download the results of a particular query. If you are new to the RW API, or want to learn more about the concept of a querying datasets, we strongly encourage you to read the [query concept](#query) documentation first. It gives you a brief and clear description of what a query is, and what it is useful for.
```

Comment:
No, it will point to the concept docs. This is hard to visualize just by reading the code, but I have ran the code and double-checked. You can see a similar situation in the current live docs by checking out #dataset and #dataset6

Reply:
Yep, this is because we have headings with equal names (`# Query` in the concept docs, and `# Query` here in the endpoint docs. The generation of the hashes for navigation gets a little crazy with this. But yep, as @tiagojsag said, this will point to the Query section on the concept docs, not this one!